gitrevise/tui.py

```diff
@@ -36,6 +36,9 @@ def build_parser() -> ArgumentParser:
         nargs="?",
         help="target commit to apply fixups to",
     )
+    parser.add_argument(
+        "pathspecs", nargs="*", help="make --cut select only from matching files"
```

Comment:
probably should set it up to only be allowed when `--cut` is used? I think clap supports this kind of thing

Reply:
yeah that was a regression, fixed